ros2_batch_job/__main__.py

```diff
@@ -431,7 +431,14 @@ def run(args, build_function):
                 exit_on_error=False)
             print('# END SUBSECTION')
 
-        blacklisted_package_names = []
+        blacklisted_package_names = [
+            'actionlib_msgs',
+            'shape__msgs',
```

Comment:
yep, thanks for catching it https://github.com/ros2/ci/pull/14/commits/452b1935e84c246532a54ac75d7924fe1dd5d9ec